ibis/omniscidb/tests/conftest.py

```diff
@@ -193,10 +237,56 @@ def temp_database(con, test_data_db: str) -> str:
     -------
     str
     """
-    name = _random_identifier('database')
-    con.create_database(name)
-    try:
-        yield name
-    finally:
-        con.set_database(test_data_db)
-        con.drop_database(name, force=True)
+    with temp_helper(con, kind='database', create=True) as ret_value:
+        yield ret_value
+
+
+@pytest.fixture
```

Comment:
so I actually don't like this temp_* at all. This is very hard to read, much better to just use: 'table', 'view', 'user' etc. These are fixtures after all.

Reply:
done it in the last commit